### DISCUSSION TITLE
Enable popup windows on cursor hover?

### DISCUSSION THREAD

**Discussion by obj-obj** [w0rp/ale#4031] [Q&A - Ask for help with problems]
I have popup windows enabled, and they show up when I run `ALEHover`, but when moving the cursor over variables and other things the info still shows in the bottom bar instead of in a popup window.

Hovering normally:
![image](https://user-images.githubusercontent.com/57120300/149409539-54f65fa4-f67f-4a7d-b21b-0401d569b563.png)

Running `ALEHover`:
![image](https://user-images.githubusercontent.com/57120300/149409596-13dd6abb-1dc1-43a4-8221-d06c5e7b9901.png)

**Comment by jsit** [comment]
Same issue here in Neovim 0.7.2. @w0rp?
